UserSettings/Search.index

```diff
@@ -0,0 +1,13 @@
+{
```

Comment:
this + Search.settings can also be .gitignore, right?

Reply:
I think the whole UserSettings folder can be ignored. Looks like it's ignored in the [gitignore gitignore](https://github.com/github/gitignore/blob/main/Unity.gitignore). I guess the gitignore in the project is an older one, so might be smart to just replace it entirely if you didn't do any personalized changes